packages/uniforms-material/src/DateField.tsx

```diff
@@ -44,7 +44,7 @@ function Date({
       fullWidth
       helperText={(error && showInlineError && errorMessage) || helperText}
       label={label}
-      InputLabelProps={{ ...labelProps, ...InputLabelProps }}
+      InputLabelProps={{ ...labelProps, ...InputLabelProps, shrink: true }}
```

Comment:
It'd be better to set is a default and let end-users decide:
```suggestion
      InputLabelProps={{ shrink: true, ...labelProps, ...InputLabelProps }}
```